developers/articles/color-scheme-definition.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: Color Sheme Definition
+title: Color Scheme Definition
 redirect_from:
-  - /Developers/Articles/Color_Sheme_Definition/
+  - /Developers/Articles/Color_Scheme_Definition/
```

Comment:
This is wrong, the redirect is there because the old website was available at that link, it needs to be kept the same.

What I meant was adding another entry here for `/developers/articles/color-sheme-definition/` (i.e. the way the page was named on the new website until your change).